packages/flutter/lib/src/material/data_table.dart

```diff
@@ -30,7 +31,7 @@ typedef DataColumnSortCallback = void Function(int columnIndex, bool ascending);
 ///
 /// One column configuration must be provided for each column to
 /// display in the table. The list of [DataColumn] objects is passed
-/// as the `columns` argument to the [new DataTable] constructor.
+/// as the `columns` argument to the new [DataTable] constructor.
```

Comment:
`[new Foo]` is the DartDoc syntax for referring to the Foo constructor

Reply:
TIL, reverting that line change.